src/main/features/claim/routes/fees.ts

```diff
@@ -3,12 +3,13 @@ import * as express from 'express'
 import { Paths } from 'claim/paths'
 import { claimAmountWithInterest } from 'app/utils/interestUtils'
 import { FeesClient } from 'fees/feesClient'
-import { Range } from 'fees/models/range'
-import { FeesTableViewHelper } from 'claim/helpers/feesTableViewHelper'
+import { FeeRange as MergableRange, FeesTableViewHelper } from 'claim/helpers/feesTableViewHelper'
 import { DraftClaim } from 'drafts/models/draftClaim'
 import { Draft } from '@hmcts/draft-store-client'
+import { FeeRange } from 'fees/models/feeRange'
+import * as config from 'config'
 
-const supportedFeeLimitInPennies: number = 1000000
+const supportedFeeLimitInGBP: number = config.get('fees.supportedFeeLimitInGBP')
```

Comment:
IMO we should make it more generic, something like `supportedClaimValue` and put his configuration entry at root level of the frontend configuration.

Another issue I see is that this only affects fee retrieval while there are other places in the code where 10K limit is imposed. I am not sure making this configurable now without updating all other places gives us much value, it might bring only more confusion giving false impression that limit is fully configurable.

I am open for discussion here.

Reply:
Agreed. Removed it from config and we'll do it later when this is changing in future.